aviatrix/resource_aviatrix_vpc.go

```diff
@@ -183,6 +183,7 @@ func resourceAviatrixVpc() *schema.Resource {
 			},
 			"resource_group": {
```

Comment:
The ordering for attributes in the schema definition map should be:
Required
Optional
Optional/Computed
Computed

So I think this needs to move up now that it is Optional/Computed